src/altinn-app-frontend/src/features/form/components/Label.test.tsx

```diff
@@ -1,78 +1,73 @@
 import * as React from 'react';
-import * as renderer from 'react-test-renderer';
+import { render } from '@testing-library/react';
 
-import Label from './Label';
+import Label, { IFormLabelProps } from './Label';
 
 describe('features > form > components >Label.tsx', () => {
-  it('should render optional', () => {
-    const rendered = renderer.create(
-      <Label
-        id='label1'
-        labelText='label.text'
-        helpText=''
-        language={{}}
-        readOnly={false}
-        required={false}
-      />,
-    );
-    expect(rendered).toMatchSnapshot();
+  let mockId: string;
+  let mockLabelText: string;
+  let mockHelpText: string;
+  let mockLanguage: any;
+  let requiredMarking: string;
+  let optionalMarking: string;
+
+  beforeEach(() => {
+    mockId =  'label1';
+    mockLabelText = 'label.text';
+    mockHelpText = '';
+    requiredMarking = '*';
+    optionalMarking = 'valgfri';
+    mockLanguage = {
+      general: {
+        optional: optionalMarking,
+      },
+      'form_filler': {
+        'required_label': requiredMarking,
+      },
+    };
+  });
+
+  test('form/components/Label.tsx -- should render default', () => {
```

Comment:
Redundant information - is described by outer `describe `
```suggestion
  test('should render default', () => {
```